app/src/ui/diff/code-mirror-host.tsx

```diff
@@ -19,6 +19,8 @@ if (__DARWIN__) {
   require('codemirror/addon/scroll/simplescrollbars')
 }
 
+require('codemirror/addon/search/search')
```

Comment:
is this a require because of something to do with how the codemirror plugin loads?

Reply:
Old habit on my end, it used to be that typescript would complain loudly if you tried to import something which didn't have a module declaration so we used require to get around that. Now it seems it only warns about it so we can switch to import instead. Resolved in a8be19d2f